etc/profile-m-z/qbittorrent.profile

```diff
@@ -10,6 +10,7 @@ noblacklist ${HOME}/.cache/qBittorrent
 noblacklist ${HOME}/.config/qBittorrent
 noblacklist ${HOME}/.config/qBittorrentrc
 noblacklist ${HOME}/.local/share/data/qBittorrent
+noblacklist ${HOME}/.local/share/qBittorrent
```

Comment:
Never blacklisted ;)

Reply:
Do you mean that I should also change `disable-programs.inc`?

Reply:
Yes